docs/Installation.md

```diff
@@ -138,15 +138,25 @@ sudo systemctl enable mariadb
 sudo systemctl start mariadb
 ```
 
-Initialize the database (unless you keep an old database):
+To initialize the database (unless you keep an old database) connect to the
+MariaDB server:
 
 ```sh
-sudo mysql < $(perl -MFile::ShareDir=dist_dir -E 'say dist_dir("Zonemaster-Backend")')/initial-mysql.sql
+sudo mysql
 ```
 
-> **Note:** This creates a database called `zonemaster`, as well as a user
-> called "zonemaster" with the password "zonemaster" (as stated in the config
-> file). This user has just enough permissions to run the backend software.
+Create the database:
+```sql
+CREATE DATABASE zonemaster;
+```
+
+Create a new user and give it all permissions on the newly created database:
+```sql
+CREATE USER 'zonemaster'@'localhost' IDENTIFIED BY 'zonemaster';
+GRANT ALL ON zonemaster.* TO 'zonemaster'@'localhost';
+```

```

Comment:
This is added complexity to the installation process. It should not be made harder to install Zonemaster-Backend.

Reply:
not related to this PR